src/main/resources/org/sagebionetworks/web/client/widget/table/v2/QueryInputViewImpl.ui.xml

```diff
@@ -22,7 +22,7 @@
 			</b:Tooltip>
 			<w:HelpWidget
 				helpMarkdown="The data contained within a Synapse `Table` can be retrieved by using a SQL-like query language either through the web portal or through the analytical clients. Queries are **not** case sensitive."
-				href="http://docs.synapse.org/articles/tables.html#table-data"
+				href="https://help.synapse.org/docs/Searching-Tables-and-Views.2009596168.html"

```

Comment:
Double-check: not in redirects?

Reply:
That's true, but #table-data is an invalid fragment on the current docs page.  
Anyway, in this context we should send the user to info about querying a Synapse Table (rather than "what is a Synapse table").

Reply:
Content is the right one (more focused than tables).